docs/examples/voting.rst

```diff
@@ -193,5 +193,9 @@ of votes.
 Possible Improvements
 =====================
 
-Currently, many transactions are needed to assign the rights
-to vote to all participants. Can you think of a better way?
+Currently, many transactions are needed to
+assign the rights to vote to all participants,
+and ``winningProposal()`` will not register there
+is a tie if two or more proposals have the same
+number of votes. Can you think of a way to fix
+these issues?
```

Comment:
```suggestion
assign the rights to vote to all participants.
Moreover, if two or more proposals have the same
number of votes, ``winningProposal()`` is not able to register
a tie. Can you think of a way to fix these issues?
```
I think this is more straightforward. What do you think?

Reply:
Yes I agree it's a bit easier to understand, will commit this suggestion